cmake/private/FairRootConfig.cmake.in

```diff
@@ -9,6 +9,7 @@
 @PACKAGE_INIT@
 
 set_and_check(@PROJECT_NAME@_PREFIX "@PACKAGE_CMAKE_INSTALL_PREFIX@")
+set_and_check(@PROJECT_NAME@_BINDIR "@PACKAGE_CMAKE_INSTALL_FULL_BINDIR@")
```

Comment:
```suggest
set(@PROJECT_NAME@_BINDIR "@PACKAGE_CMAKE_INSTALL_FULL_BINDIR@")
```

Let's only use `set_and_check` on the prefix or pathes that are guaranteed to be there for devel package trees. I have complicated packaging for myself in the past by being "too generous" with the `set_and_check`.

Reply:
I agree, we would only unlikely run into problems with the `BINDIR` at the moment, still I would prefer if we would not use the check here. Thx!